src/mock-data/mockChallenges.js

```diff
@@ -1,6 +1,8 @@
 export const mockChallenges = {
```

Comment:
This should be MOCK_CHALLENGES. Someone needs to change it. Please ensure all files using this is changed as well

Reply:
I've updated this. @jagadeepavula I also updated this in the ChallengePage.